_posts/2013-07-17-using-caption-services-with-html5-video.md

```diff
@@ -22,10 +31,12 @@ So how do we pull off putting captions into a video after the video has been pro
         <track src="your-video-transcript.vtt" label="English Captions" kind="subtitles" srclang="en-us" default />
     </video>
 
+Many frameworks built on top of HTML5 video make it even simpler to add in captions, like [Video.js](https://videojs.com/).
```

Comment:
replace 
"frameworks built on top of HTML5 video" 
with 
"frameworks built on top of the video element"